std_msgs/Bool get_list
std_msgs/Bool set
std_msgs/Bool show_current
std_msgs/Bool remove

std_msgs/String cs_name
std_msgs/Int16 secs

---

std_msgs/Bool success
std_msgs/String message
std_msgs/String[] cs_list